### PR TITLE
CEO PR#1 - Leaders/CEOs Variant - CardName, CardType, LeadersExtension, LeaderCard

### PR DESCRIPTION
**Pull request by d-little**
Init the CEO/Leader extension/expansion/variant

[`LeadersExtension.ts`](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:leadersVariant-PR?expand=1#diff-c8db251c4b1c2bb96691f5e6f1878a01eb401e356f2ff28344180391bafebefc) contains two functions that reference cards that _will_ be implemented: `calculateVictoryPoints()` + `getBonusWildTags()` ( for `Duncan` & `Xavier`  respectively)

We can remove these two from this initial PR if necessary and only include them when pushing in Xavier + Duncan in future PRs

### CODE REVIEW

**Comment by kberg** [comment]
https://github.com/terraforming-mars/terraforming-mars/actions/runs/3867260623/jobs/6591920280 this doesn't compile.

**Comment by d-little** [comment]
> 

Fix it up, DebugUI.vue and [LogPanel.vue](https://github.com/terraforming-mars/terraforming-mars/pull/5209/commits/41375f2716bffe20efd6ea7705e9974f25cd6020#diff-2336a20ae30e43df8e99a116367de6ba91c6d028b79757fd397e2e1bdc2dbdda) both wanted a couple of properties set

**Review by kberg — COMMENTED** [review]
Pretty much there. Thank you!

File: src/server/cards/leaders/LeaderCard.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {Player} from '../../Player';
+import {IProjectCard} from '../IProjectCard';
+import {CardType} from '../../../common/cards/CardType';
+import {ICard} from '../ICard';
+
+export interface LeaderCard extends IProjectCard {
+  isDisabled?: boolean;
+  opgActionIsActive?: boolean;
+  generationUsed?: number;
+  onTRIncrease?: (player: Player) => undefined;
```
[the remark this answers is not in the document]
Comment (kberg) [reply src/server/cards/leaders/LeaderCard.ts]: Looks like I was able to commit changes directly to this PR. I did that in order to add comments, but now I also removed the function. Hope that's OK.

**Review by kberg — APPROVED** [review]
This is a good change. Thank you for starting it!

**Pull request merged**